### PR TITLE
Changed the order of the command_args an put some space in it

### PR DESCRIPTION
**Pull request by str4el**
'-d/some/file' isn't working but '-d /some/file' does, so this is
a workaround. Also global options should come first.

This was discussed here [#277](https://github.com/knxd/knxd/pull/277)

### CODE REVIEW

File: contrib/gentoo/sys-apps/knxd/files/knxd.init [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 pidfile="/run/knxd/knxd-@SLOT@.pid"
 command="/usr/bin/knxd"
-command_args="${KNXD_OPTS} --pid-file=${pidfile} -d${KNXD_LOGFILE}"
+command_args="--pid-file=${pidfile} -d ${KNXD_LOGFILE} ${KNXD_OPTS}"
```
Comment (smurfix) [inline contrib/gentoo/sys-apps/knxd/files/knxd.init]: Please use the long form, `--daemon=FILENAME`.

Comment (str4el) [inline contrib/gentoo/sys-apps/knxd/files/knxd.init]: I was, but it doesn't work either

Comment (smurfix) [inline contrib/gentoo/sys-apps/knxd/files/knxd.init]: Apparently that's either a bug in the argp library or I don't know how to use that correctly.
Use `--daemon FILENAME`.
I will investigate further.

Comment (str4el) [inline contrib/gentoo/sys-apps/knxd/files/knxd.init]: I can  change it, in the proper way after the bugfix #280 is merged

Comment (smurfix) [inline contrib/gentoo/sys-apps/knxd/files/knxd.init]: done

**Pull request merged**